src/org/stringtemplate/v4/Interpreter.java

```diff
@@ -803,9 +803,10 @@ protected int writePOJO(STWriter out, InstanceScope scope, Object o, String[] op
         return n;
     }
 
-    private <T> String renderObject(InstanceScope scope, String formatString, T o, Class<? extends T> attributeType) {
+    @SuppressWarnings({ "rawtypes", "unchecked" })
+    private String renderObject(InstanceScope scope, String formatString, Object o) {
```

Comment:
```suggestion
    private <T> String renderObject(InstanceScope scope, String formatString, Object o, Class<T> attributeType) {
```

Reply:
Please also adapt the call to add the argument for the new parameter.

Reply:
And shouldn't it be `T o`?

Reply:
> Please also adapt the call to add the argument for the new parameter.

The call site is already fine.

> And shouldn't it be `T o`?

In theory, yes, but in practice Java's type inference can't figure that out.